draft-kalos-bbs-per-verifier-linkability.md

```diff
@@ -217,11 +217,40 @@ This section will provide a high level description of the required operations, b
 Initially, the Prover will chose a set of messages `committed_messages` that they want to be included in the signature, without reveling them to the Signer. They will also choose their part of the pseudonym secret `prover_nym` as a random scalar value.
 
 ```
-1. committed_messages.append(prover_nym)
-2. (commitment_with_proof, secret_prover_blind) = Commit(
+(commitment_with_proof, secret_prover_blind) = Commit(
                                                    committed_messages,
+                                                   nym_secret,
```

Comment:
We are committing to *prover_nym* here. *nym_secret* is not computed yet. This is for initial prover to signer communications.